flask/app/models.py

```diff
@@ -252,7 +254,7 @@ class Dataset(db.Model):
     )
     notes: str = db.Column(db.Text)
     condition: DatasetCondition = db.Column(db.Enum(DatasetCondition), nullable=False)
-    extraction_protocol: str = db.Column(db.String(100))
+    extraction_protocol: str = db.Column(db.String(150))
```

Comment:
Where did this change come from?

Reply:
I thought I saw it was a length of 150 at some point in the gene pull migration. I believe it was carried over by setting `compare_type = True` in `context.configure()`. Anyways I can reset it to a length of 100?